### ISSUE TITLE
Message terminate not sent

### ISSUE THREAD

**Issue by c-rom** [metajack/strophejs#21]
When I try to disconnect, the message "terminate" is not sent.
I call this :
     connection.flush();
     connection.disconnect();

Some requests still to be processed. I have done a hack for me in "_throttledRequestHandler:"

```
    if (this._requests.length > 0) {
    for(i=0; i<this._requests.length; i++)
            this._processRequest(i);
    }
```

I don't know well the strophejs framework so maybe there is a clean way to do this?

Issue appears in Google Chrome on Ubuntu

**Comment by metajack** [comment]
When is the disconnect() being called? If it's during onUnload, then it's not going to work, as async requests won't work reliably in that callback.

**Comment by santiago** [comment]
Hi

I'm having exactly same issue. I'm just calling .disconnect(), not within unLoad, and I don't even see (in Firebug) that something is sent to the connection.
It is the timeout which finally gets the user offline.

Btw. Where do I get latest release? Not available on Home Page, thou I've got 1.0.1, I downloaded it more than 6 months ago.
